lib/addons/OPA/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ClusterAddOn, ClusterInfo } from "../../stacks/cluster-types";
+
+export class OpaAddOn implements OpaAddOn {
+
+    deploy(clusterInfo: ClusterInfo): void {
```

Comment:
Please take a look at the supported values.yml for the chart and see if any of those values make sense to be passed explicitly (e.g. namespace, service type). If not, then all values should be allowed similar to what we did in for example Nginx addon (see NginxAddOnProps.values) so that customer could tune it, e.g. setting whether deny attempts are logged, etc.